src/hello-world-app.js

```diff
@@ -1,6 +1,7 @@
 import '@brightspace-ui/core/components/button/button.js';
 import '@brightspace-ui/core/components/demo/code-view.js';
 import '@brightspace-ui/core/components/dialog/dialog-confirm.js';
+import '@brightspace-ui/core/components/dialog/dialog-fullscreen';
```

Comment:
I've seen this not work in certain browsers/build systems, so better safe than sorry.

```suggestion
import '@brightspace-ui/core/components/dialog/dialog-fullscreen.js';
```

Reply:
Yep, this was a typo 😆